lib/rbs/unit_test/spy.rb

```diff
@@ -128,7 +128,7 @@ def wrapped_object
                 end
               end.ruby2_keywords
             )
-          end.new()
+          end.allocate()
```

Comment:
@sampersand Can you explain why you use `allocate` instead of `new`?

Reply:
It's a minor bug. TL;DR: if you you pass `initialize` as the name of the method, `.new()` will call it with zero params. By usign `.allocate()`, we can safely mock `#initialize()` methods.

Reply:
Got it. I'm not against for using `.allocate` here, but I also recommend using `.new` instead of `#initialize` to test.